packages/bits/src/lib/toolbar/toolbar-keyboard.service.ts

```diff
@@ -0,0 +1,100 @@
+import { Injectable } from "@angular/core";
+import { KEYBOARD_CODE } from "../../constants";
+import { MenuComponent } from "../menu";
+
+@Injectable()
+export class ToolbarKeyboardService {
+    public dynamicContainer: HTMLElement | null;
+
+    public moreBtn: HTMLButtonElement | null;
+
+    public menuComponent: MenuComponent | null = null;
+
+    public initService(container: HTMLElement, menu: MenuComponent | null): void {
+        this.dynamicContainer = container.querySelector(".nui-toolbar-content__dynamic");
+
+        if (this.dynamicContainer) {
+            this.moreBtn = this.dynamicContainer.querySelector(".nui-button.btn.menu-button");
+        }
+
+        if (menu) {
+            this.menuComponent = menu;
+        }
+    }
+
+    public disableFocusForMoreBtn(): void {
```

Comment:
This name would read better to me. Just a nit:

```suggestion
    public disableMoreBtnFocus(): void {
```